.github/workflows/elasticsearch.yml

```diff
@@ -2,26 +2,18 @@ name: elasticsearch
 
 on:
   workflow_dispatch:
+  push:
+    branches: [ master ]
 
 jobs:
-  elasticsearch7:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v3
-    - uses: actions/setup-java@v3
-      with:
-        distribution: 'temurin'
-        java-version: '17'
-        cache: 'maven'
-    - run: export SEARCH_ENGINE_DOCKERFILE="docker/elasticsearch/Dockerfile"; mvn clean install
   elasticsearch8:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
-    - uses: actions/setup-java@v3
-      with:
-        distribution: 'temurin'
-        java-version: '11'
-        cache: 'maven'
-    - run: export SEARCH_ENGINE_DOCKERFILE="docker/elasticsearch8/Dockerfile"; mvn clean install
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+          cache: 'maven'
+      - run: export SEARCH_ENGINE_DOCKERFILE="docker/elasticsearch8/Dockerfile"; mvn clean install
```

Comment:
maybe we can just rename folder: elasticsearch8 to elasticsearch as folder elasticsearch is removed now

Reply:
I think it's okay to leave as is in case we will need to support more then one version in future